data/pom.xml

```diff
@@ -6,7 +6,11 @@
     <artifactId>kusto-data</artifactId>
     <name>Kusto data client library</name>
     <description>Kusto client library for executing queries and retrieving data</description>
-
+    <url>https://github.com/Azure/azure-kusto-java/tree/master/data</url>
+    <scm>
+        <url>scm:git:https://github.com/Azure/azure-kusto-java</url>
```

Comment:
This is an interesting use case. We have 2 artifacts, both within the same git repo.

- What use case does this change come to address?
- Does this git URL achieve that intended goal, considering that it doesn't quite point to the repo specific to the artifact?

Reply:
urls here are wrong
https://search.maven.org/artifact/com.microsoft.azure.kusto/kusto-data/2.4.1/jar